billiard/pool.py

```diff
@@ -441,7 +441,7 @@ def _make_recv_method(self, conn):
             if hasattr(conn, 'get_payload') and conn.get_payload:
                 get_payload = conn.get_payload
 
-                def _recv(timeout, loads=pickle_loads):
+                def _recv(timeout, loads=dill_loads):
```

Comment:
can you please update or add some tests to check the dill usage?

Reply:
Could you specify what sort of test you would like added? Pool instanciation is already tested, as well as messages sending and receiving (which should handle the tests on whether this would introduce side effects or not). I have added some tests following this review, but I don't know if they meet your expectations.

Reply:
Is this resolved?

Reply:
Can I resolve this conversation?